runtime/oti/j9consts.h

```diff
@@ -648,6 +648,12 @@ extern "C" {
 #define J9_THREAD_START_UNSPECIFIED_ERROR 0x5
 #define J9_THREAD_START_THROW_CURRENT_EXCEPTION 0x6
 
+#if JAVA_SPEC_VERSION >= 19
+#define J9_VIRTUALTHREAD_INTERNAL_STATE_NONE      0x0
+#define J9_VIRTUALTHREAD_INTERNAL_STATE_SUSPENDED 0x1
+#define J9_VIRTUALTHREAD_INTERNAL_STATE_CARRIERID_MASK (~(uintptr_t)0xff)
```

Comment:
Do we know that `J9VMThread` objects are allocated on 256 byte boundaries?
Since it seems we're only using the least significant bit for the state, the mask could be `(~(uintptr_t)0x1)`.

Reply:
All J9VMThread are aligned using `J9VMTHREAD_ALIGNMENT` which is `0x100`
Using `0xff` would indicate that the rest bits are reserved and safe for future use.

Reply:
Thank you. A comment relating `J9_VIRTUALTHREAD_INTERNAL_STATE_CARRIERID_MASK` and `J9VMTHREAD_ALIGNMENT` would be welcome, or even
```c
#define J9_VIRTUALTHREAD_INTERNAL_STATE_CARRIERID_MASK (~(uintptr_t)(J9VMTHREAD_ALIGNMENT - 1))
```
but then `J9_VIRTUALTHREAD_INTERNAL_STATE_xxx` ought to move to `j9nonbuilder.h`.